apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -16,19 +16,34 @@ import (
 
 type bundleSuite struct {
 	coretesting.BaseSuite
-	facade bundle.Bundle
+	auth     *apiservertesting.FakeAuthorizer
+	facade   *bundle.APIv2
+	st       *mockState
+	modelTag names.ModelTag
 }
 
 var _ = gc.Suite(&bundleSuite{})
 
 func (s *bundleSuite) SetUpTest(c *gc.C) {
 	s.BaseSuite.SetUpTest(c)
-	auth := apiservertesting.FakeAuthorizer{
+	s.auth = &apiservertesting.FakeAuthorizer{
 		Tag: names.NewUserTag("who"),
 	}
-	facade, err := bundle.NewBundle(auth)
+
+	s.st = newMockState()
+	s.modelTag = names.NewModelTag("some-uuid")
+
+	s.facade = s.makeAPI(c)
+}
+
```

Comment:
We probably want a test that makes a v1 api so we can show that ExportBundle isn't exposed.